.travis.yml

```diff
@@ -1,21 +1,18 @@
+sudo: false
 language: python
-python:
-  - "3.6"
-  - "3.7"
-  - "3.8"
-before_script:
-  # stop the build if there are Python syntax errors or undefined names
-  - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-  # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
-  - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-# command to run tests
-script: coverage run --branch --source=libmodernize setup.py test
-# Ensure dependencies are installed
+matrix:
+  include:
+    - python: "3.6"
+      env: TOXENV=py36,lint,coveralls
+    - python: "3.7"
+      env: TOXENV=py37,lint,coveralls
+    - python: "3.8"
+      env: TOXENV=py38,lint,coveralls
 install:
-  - pip install .
-  - pip install 'coveralls==1.8.2'
-  - pip install flake8
-  - pip install 'six >= 1.13.0'
-after_success:
-  coveralls
-sudo: false
+  - nvm install 14
+  - pip install tox
+script: tox
+cache:
+  pip: true
+  directories:
+    - $HOME/.cache/pre-commit
```

Comment:
Would you be interested in switching from Travis to Github Actions? It would enable testing on Linux, macOS, and Windows to help find some of the subtle bugs and incompatibilities on alternate OSs.  I'd be happy to put that together for you.

Also, I know you have pre-commit set up, but probably worth adding a step to the CI that verifies that all files are correctly formatted and sorted to help catch cases where PRs are submitted by users that haven't set up pre-commit.

Reply:
> Would you be interested in switching from Travis to Github Actions? It would enable testing on Linux, macOS, and Windows to help find some of the subtle bugs and incompatibilities on alternate OSs. I'd be happy to put that together for you.

I'm completely in favour of removing travis and replace with GHA calling tox

> Also, I know you have pre-commit set up, but probably worth adding a step to the CI that verifies that all files are correctly formatted and sorted to help catch cases where PRs are submitted by users that haven't set up pre-commit.

In this PR `tox -e lint` calls `pre-commit`